### ISSUE TITLE
Add term abnormal metacarpal ossification

### ISSUE THREAD

**Issue by lisannesw** [obophenotype/human-phenotype-ontology#10413]
**Preferred term label:**
Abnormal metacarpal ossification

**Synonyms**
Abnormal ossification involving metacarpal bones - Abnormal maturation of long bone of hand

**Definition (free text, please give PubMed ID)**
Any abnormal process of ossification of the metacarpal bones, which normally are each ossified from two centers: one for the body, and one for the head (metacarpal II,III,IV, and V) and one for the body and one for the base (metacarpal I). At the eighth to ninth week of fetal life, ossification commences in the middle of the body. At the third year the distal extremities of the metacarpals of the fingers and the base of the metacarpal of the thumb begin to ossify; they unite with the bodies at about the twentieth year
Gray's Anatomy. Churchill Livingstone. (2011) ISBN:0443066841

**Parent term (use hpo.jax.org/app)**
Abnormal metacarpal morphology HP:0005916
Abnormal hand bone ossification HP:0010660

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
For example: Acrocapitofemoral dysplasia (omim607778) PMID: 12624140
Acrodysostosis 2 with or without hormone resistance (omim614613) PMID: 36006853
Acromesomelic dysplasia type-grebe (omim200700) PMID: 27577507

**Your nano-attribution (ORCID)**
Andreas Zankl: 0000-0001-8612-1062, @azankl
Lisanne Wit: 0009-0007-1020-5412

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6000651
- label: Abnormal metacarpal ossification
- parents: [Abnormal metacarpal morphology (HP:0005916)](https://hpo.jax.org/app/browse/term/HP:0005916) | [Abnormal hand bone ossification (HP:0010660)](https://hpo.jax.org/app/browse/term/HP:0010660)
- pmid: PMID:12624140
- def: Any abnormal process of ossification of the metacarpal bones, which normally are each ossified from two centers: one for the body, and one for the head (metacarpal II,III,IV, and V) and one for the body and one for the base (metacarpal I). At the eighth to ninth week of fetal life, ossification commences in the middle of the body. At the third year the distal extremities of the metacarpals of the fingers and the base of the metacarpal of the thumb begin to ossify; they unite with the bodies at about the twentieth year.
- comment: 
- orcid: orcid:0000-0001-8612-1062
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10413
- synonyms: Abnormal maturation of long bone of hand(NONE)|Abnormal ossification involving metacarpal bones(NONE)
